rest_api/config.py

```diff
@@ -23,8 +23,8 @@
 NAME_FIELD_NAME = os.getenv("NAME_FIELD_NAME", "name")
 SEARCH_FIELD_NAME = os.getenv("SEARCH_FIELD_NAME", "text")
 FAQ_QUESTION_FIELD_NAME = os.getenv("FAQ_QUESTION_FIELD_NAME", "question")
-EMBEDDING_FIELD_NAME = os.getenv("EMBEDDING_FIELD_NAME", None)
-EMBEDDING_DIM = os.getenv("EMBEDDING_DIM", None)
+EMBEDDING_FIELD_NAME = os.getenv("EMBEDDING_FIELD_NAME", "embedding")
+EMBEDDING_DIM = os.getenv("EMBEDDING_DIM", 768)
```

Comment:
I think we should cast EMBEDDING_DIM to int here

Reply:
done